pkg/controller/wavefrontcollector/wavefrontcollector_controller.go

```diff
@@ -245,6 +246,41 @@ func newPodSpecForCR(instance *wavefrontv1alpha1.WavefrontCollector) corev1.PodS
 		debug = "--log-level=debug"
 	}
 
+	volumeMounts := []corev1.VolumeMount{{
+		Name:      "procfs",
+		MountPath: "/host/proc",
+		ReadOnly:  true,
+	}}
+
+	volumes := []corev1.Volume{{
+		Name: "procfs",
+		VolumeSource: corev1.VolumeSource{
+			HostPath: &corev1.HostPathVolumeSource{
+				Path: "/proc",
+			},
+		},
+	}}
+
+	if !instance.Spec.UseOpenshiftDefaultConfig {
```

Comment:
should this be :
if !instance.Spec.Openshift && !instance.Spec.UseOpenshiftDefaultConfig { ... }